devtools/platform-descriptor-json/src/main/resources/templates/README.ftl

```diff
@@ -0,0 +1,34 @@
+# ${project_artifactId} project
+
+This project uses the Supersonic Subatomic Java Framework named Quarkus.
+
+If you want to learn more about this framework, please check this website: https://quarkus.io
+
+
+## Running the application in dev mode
+
+You can run your application in dev mode that will allow to have hot reload feature using:
+```
+./mvnw compile quarkus:dev
```

Comment:
Just to be sure as this is completely Gradle specific, this file is not generated for Gradle projects?

Reply:
hmmm... you're right, I too late thought about Gradle after the PR creation.

Could you tell me how to have 2 templates ? (one for Gradle and another for Maven)

I'll check that tonight.

Reply:
done